requirements.in

```diff
@@ -0,0 +1,321 @@
+absl-py==2.2.2
```

Comment:
Ignore this file. Depends on PR:
https://github.com/seintun/proj_pi_server/pull/15

I'm waiting until the other PRs are approved before merging mines, since I'm good at dealing with or avoiding merge conflicts